tests/test_runner/test_discover_runner.py

```diff
@@ -294,6 +294,12 @@ def test_excluded_tags_displayed(self):
             runner.build_suite(['test_runner_apps.tagged.tests'])
             self.assertIn('Excluding test tag(s): bar, foo.\n', stdout.getvalue())
 
+    def test_no_of_tests_displayed(self):
+        runner = DiscoverRunner(verbosity=1)
```

Comment:
Since `verbosity=1` is the default, you can leave this out. (It's good to be testing the default behavior.)